manifests/0000_09_service-ca-operator_02_crd.yaml

```diff
@@ -3,12 +3,116 @@ kind: CustomResourceDefinition
 metadata:
   name: servicecas.operator.openshift.io
 spec:
-  scope: Cluster
   group: operator.openshift.io
-  version: v1
   names:
     kind: ServiceCA
     plural: servicecas
     singular: serviceca
-  subresources:
-    status: {}
+  scope: Cluster
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            logLevel:
+              description: logLevel is an intent based logging for an overall component.  It
+                does not give fine grained control, but it is a simple way to manage
+                coarse grained logging choices that operators have to interpret for
+                their operands.
+              type: string
+              enum:
+                - 'Normal'
```

Comment:
The default operator config sets logLevel to "4", and the controller code expects that. We should probably allow that value for the time being.

Reply:
I see, I'm going to remove the `enum`s then, I was under the impression the logLevel would _always_ be 1 of the 4 listed in api/operator/v1/types.go, but since this isn't the case, we should leave out the enums altogether, I think.